modules/installation-user-defined-tags-azure.adoc

```diff
@@ -0,0 +1,109 @@
+
+You can use the `.platform.azure.userTags` field in the `install-config.yaml` file to define the list of user-defined Azure tags. You must set the `featureSet` field as `TechPreviewNoUpgrade` in the `install-config.yaml` file.
+
+The user-defined Azure tags have the following limitations:
+
+* A tag key can have a maximum of 128 characters.
+* A tag key must begin with a letter, end with a letter, number or underscore, and can contain only letters, numbers, underscores, periods, or hyphens.
+* Tag keys are case-insensitive.
+* Tag keys cannot be `name`. It cannot have prefixes such as `kubernetes.io`, `openshift.io`, `microsoft`, `azure`, `windows`.
+* A tag value can have a maximum of 256 characters.
+* You can configure a maximum of 10 tags for resource group and resources.
+
+.Sample `install-config.yaml` file
+[source,yaml]
+----
+additionalTrustBundlePolicy: Proxyonly <1>
+apiVersion: v1
+baseDomain: catchall.azure.devcluster.openshift.com <2>
+featureSet: TechPreviewNoUpgrade <3>
+compute:
+- architecture: amd64
+  hyperthreading: Enabled
+  name: worker
+  platform: {}
+  replicas: 3
+controlPlane:  <4>
+  architecture: amd64
+  hyperthreading: Enabled
+  name: master
+  platform: {}
+  replicas: 3
+metadata:
+  creationTimestamp: null
+  name: bhb <5>
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.0.0.0/16
+  networkType: OVNKubernetes
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  azure:
+    baseDomainResourceGroupName: os4-common <6>
+    cloudName: AzurePublicCloud <7>
+    outboundType: Loadbalancer
+    region: southindia <8>
+    userTags:
+      createdBy: bhb <9>
+      environment: dev <10>
+----
+<1>
+<2>
+<3>
+<4>
+<5>
+<6>
+<7>
+<8>
+<9>
+<10>
+
+After creating the cluster, You can refer to the `.status.platformStatus.azure.resourceTags` in the `config.openshift.io/Infrastructure` resource `cluster` object to know the list of the defined tags.
+
+. Sample ``install-config.yaml file
+[source,terminal]
+----
+apiVersion: config.openshift.io/v1
+kind: Infrastructure
+metadata:
+  name: cluster <1>
+spec:
+  platformSpec:
+    type: Azure
+status:
+  infrastructureName: bhb-2497l <2>
+  platform: Azure
+  platformStatus:
+    azure:
+      resourceTags: <3>
+      - key: environment <4>
+        value: dev <5>
+    type: Azure <6>
+----
+<1>
+<2>
+<3>
+<4>
+<5>
+<6>
+
+By default, {prodcut-title} installer attaches the {prodcut-title} tags to the Azure resources. These {prodcut-title} tags are not available for the users. The format of  {product-title} tags is `kubernetes.io_cluster.<cluster_id>:owned`. The `cluster_id` parameter is the value of `.status.infrastructureName` present in `config.openshift.io/Infrastructure`.
```

Comment:
nit: {prodcut-title} to be replaced with {product-title}

Reply:
Addressed